README.md

```diff
@@ -28,15 +28,15 @@ Install on your own machine:
 Get Involved:
 
 - [Contribute Code](http://docs.mattermost.com/developer/contribution-guide.html)
-- [Find "Help Wanted" projects](https://mattermost.atlassian.net/issues/?filter=10101)
+- [Find "Help Wanted" projects](https://mattermost.atlassian.net/browse/PLT-7805?filter=10101&jql=status%20in%20(Open%2C%20Reopened)%20AND%20labels%20in%20(accepting-pull-requests%2C%20help-wanted))
```

Comment:
This doesn't look right.

Reply:
ya it was quite difficult to make the tags work properly, is there a "short url" format that can be exported from jira?